src/test/java/uk/gov/hmcts/reform/cdapi/service/impl/CrdServiceImplTest.java

```diff
@@ -60,8 +61,6 @@ void retrieveCategoriesByCategoryId() {
         assertEquals(listOfValueDtos.get(0).getHintTextCy(), actualCategory.getHintTextCy());
         assertEquals(listOfValueDtos.get(0).getHintTextEn(), actualCategory.getHintTextEn());
         assertEquals(listOfValueDtos.get(0).getLovOrder(), actualCategory.getLovOrder());
-        assertEquals(listOfValueDtos.get(0).getServiceId(), actualCategory.getServiceId());
-        assertEquals(listOfValueDtos.get(0).getActive(), actualCategory.getActiveFlag());
         assertNull(actualCategory.getChildNodes());
     }
 
```

Comment:
Add test cases for below scenarios:
1. request call with category id and service id with no child records
2. request call with category id and service id with child records

Reply:
Thanks @rajkatla-hmcts I have done with unit test cases its working fine